### PR TITLE
Dp plan a - add plan A

### PR DESCRIPTION
**Pull request by d34d-p00l**
This adds plan A to the master plan and closes issue #5 
@i-marsh please review

💯 feel this plan will work

### CODE REVIEW

File: plan-a.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+### Initial Plan to save Earth
+
+This is a **cool** plan
+- [ ] Put the team together
+  - Wolverine
+  - Superman
+  - Flash
+  - Green Arrow
```
Comment (i-marsh) [inline plan-a.md]: This cannot happen @d34d-p00l 
There is no crossover between DC and Marvel...
need to rethink your team strategy

**Review by i-marsh — CHANGES_REQUESTED** [review]
Please make the team Marvel only

File: plan-a.md (hunk 1 above)
Comment (d34d-p00l) [reply plan-a.md]: Doh .. headslap ... I'll fix it

**Review by i-marsh — APPROVED** [review]
awesome job thanks :tada:

**Pull request merged**